src/components/Home.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
+import '../../public/css/style.css';
```

Comment:
Don't import any CSS file here. The compiled CSS file should be imported in the entry point of the project. i.e, `App.js`